eclair-node/src/test/scala/fr/acinq/eclair/api/ApiServiceSpec.scala

```diff
@@ -220,6 +221,26 @@ class ApiServiceSpec extends AnyFunSuite with ScalatestRouteTest with IdiomaticM
       }
   }
 
+  test("'channelbalances' asks relayer for current balances from all channels even if disabled") {
```

Comment:
Actually this test doesn't verify that we ask the relayer for current balances (and the previous test doesn't either).
This test and the previous one only verify the JSON formatting.
It's worth renaming them to clarify this and adding tests to `EclairImplSpec.scala` to check that we ask the relayer for the right information.

Reply:
I got a little side tracked trying to verify the data returned by the `relayer` actor. But then I realized it's enough to just verify that eclair asks the `relayer` for channels with the right messages in `EclairImplSpec.scala`. It's up to the test in `ChannelRelayerSpec.scala` called "get outgoing channels" to exercise the `isEnabled` logic.

If I understand this correctly, fixed in [e8f746d](https://github.com/ACINQ/eclair/pull/2196/commits/e8f746d3c116e7a15914f583252fbc32785e5336).

Reply:
> > @t-bast How should I document this new call in the API documentation? should documentation be part of this PR too?
> 
> I'll add to the release notes the new `channelbalances` documentation so @t-bast can [update the API docs](https://github.com/ACINQ/eclair/blob/master/BUILD.md#build-the-api-documentation) when 0.71 is released.

fixed in [cdd0e38](https://github.com/ACINQ/eclair/pull/2196/commits/cdd0e385054ef253c0f26ff1a972a6ab72924730)